frontend/components/FleetMarkdown/FleetMarkdown.tsx

```diff
@@ -15,8 +15,8 @@ interface ICustomLinkProps {
   newTab?: boolean;
 }
 
-const CustomLink = ({ text, href, newTab = false }: ICustomLinkProps) => {
```

Comment:
This uses `React.Node` type for `text`, is there a reason it's not `string`? The CustomLink component I built uses string so I'm looking if we can consolodate.

Reply:
Zoomed